x-pack/plugins/observability_onboarding/public/components/app/custom_logs/configure_logs.tsx

```diff
@@ -423,7 +423,7 @@ export function ConfigureLogsContent() {
                   helpText={
                     <FormattedMessage
                       id="xpack.observability_onboarding.configureLogs.customConfig.helper"
-                      defaultMessage="Here YAML configuration options can be used to be added to your configuration. Be careful using this as it might break your configuration file. {learnMoreLink}"
+                      defaultMessage="Add YAML configuration options to your agent configuration. Be careful using this feature as it can break your configuration file. {learnMoreLink}"
```

Comment:
Might be helpful to include some info on why a user would want to add configs here. Also, I'm not sure this is the best link.

Reply:
The docs we should link to here would be around using multilline, or how to exclude log lines. Some links below, unfortunately pointing to filebeat:

* https://www.elastic.co/guide/en/beats/filebeat/current/multiline-examples.html
* https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#_scanner_options